tests/__init__.py

```diff
@@ -35,3 +35,17 @@ def get_containers_by_name(doc, include_init_containers=False) -> dict:
         )
 
     return c_by_name
+
+
+def get_cronjob_containerspec_by_name(doc) -> dict:
+    """Given a single doc, return all the containers by name.
+
+    doc must be a valid spec for a pod manager. (EG: ds, sts)
```

Comment:
Is this comment right? AFAIK only cronjobs have a jobTemplate, and the function name specifies cronjob.

```
$ git grep -l jobTemplate tests/k8s_schema/
tests/k8s_schema/v1.25.0-standalone/cronjob-batch-v1.json
tests/k8s_schema/v1.26.0-standalone/cronjob-batch-v1.json
tests/k8s_schema/v1.27.0-standalone/cronjob-batch-v1.json
tests/k8s_schema/v1.28.0-standalone/cronjob-batch-v1.json
tests/k8s_schema/v1.29.0-standalone/cronjob-batch-v1.json
tests/k8s_schema/v1.30.0-standalone/cronjob-batch-v1.json
```

```suggestion
    doc must be a valid spec for a CronJob.
```